src/pages/integrations/gift-cards/confirm-card-purchase/confirm-card-purchase.html

```diff
@@ -0,0 +1,97 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>Confirm</ion-title>
+    <ion-buttons right>
+      <button (click)="cancel()" ion-button>
+        Cancel
```

Comment:
needs translation

Reply:
In view of forgetting the encryption key, I felt the need to delete the wallet in order to import it again and not to place an encryption key. When trying to import the wallet using the twelve words the system throws a message Can not access the wallet from the server. Please verify: The password of the recovery phrase (if it exists). The derivation route. URL address of the Wallet Service. And also in a separate note the message says: To import a wallet from third-party software you must go to Add wallet Create wallet and enter recovery phrase. I do not know what to do, you can tell me I need to use the funds deposited there urgently. Thank you I hope your help